.coderabbit.yaml

```
@@ -0,0 +1,38 @@
+  high_level_summary: true
+  poem: false
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: false
+  instructions: >-
+    # Code Review Instructions
+
+    - Ensure code follows best practices and coding standards.
+    - For **CPP** code, follow 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)